boards/imix/src/main.rs

```diff
@@ -102,6 +103,7 @@ static mut CHIP: Option<&'static sam4l::chip::Sam4l<Sam4lDefaultPeripherals>> =
 #[link_section = ".stack_buffer"]
 pub static mut STACK_MEMORY: [u8; 0x2000] = [0; 0x2000];
 
+/// Supported drivers by the platform
```

Comment:
Remove

Reply:
I can't remove this because every struct requires a comment before it to define the struct.

Reply:
There wasn't a comment before this PR, but the formatter must have passed for that code to reach master

Reply:
Maybe the formatted updated? That happens...

Reply:
It's fine to keep it.